prehook/richcontainer/systemd_launcher_linux.go

```diff
@@ -43,6 +45,29 @@ func (l *systemdLauncher) Launch(opt *prehook.HookOptions, spec *specs.Spec) err
 
 	cmd := spec.Process.Args
 
+	if len(cmd) == 0 {
+		return errors.New("not set cmd")
```

Comment:
Could you make this error more readable?
Such as `no cmd set in process of container runtime spec`